src/lib/y2storage/storage_manager.rb

```diff
@@ -278,20 +278,21 @@ def committed?
       @committed
     end
 
-    # Performs in the system all the necessary operations to make it match the
-    # staging devicegraph.
+    # Performs in the system all the necessary operations to make it match the staging devicegraph.
     #
     # Beware: this method can cause data loss
     #
-    # The user is asked whether to continue on each error reported by
-    # libstorage-ng.
+    # The user is asked whether to continue on each error reported by libstorage-ng.
     #
-    # @return [Boolean] whether commit was successfull, false if libstorage-ng
-    #   found a problem and it was decided to abort due to that
-    def commit(force_rw: false)
+    # @param force_rw [Boolean] if mount points should be forced to have read/write permissions.
+    # @param callbacks [Y2Storage::Callbacks::Commit]
+    #
+    # @return [Boolean] whether commit was successful, false if libstorage-ng found a problem and it was
+    #   decided to abort.
+    def commit(force_rw: false, callbacks: nil)
```

Comment:
question is if you want here to allow passing explicit nil or not. If not, then you can add this call to method signature like `def commit(force_rw: false, callbacks: Callbacks::Commit.new)` and it is still backward compatible.

Reply:
Well, libstorage-ng does not force to pass a callbacks, so maybe in the future we decide to allow `nil` (although is not very likely).